apps/client/src/atoms/EditButton.tsx

```diff
@@ -2,10 +2,14 @@ import React from "react";
 
 import { EditButtonStyle } from "./EditButtonStyle";
 
-const EditButton: React.FC<React.HTMLAttributes<HTMLElement>> = ({
-  onClick,
-}) => (
-  <EditButtonStyle {...{ onClick }} variant="textButton">
+const EditButton: React.FC<
+  { editDisabled: boolean } & React.HTMLAttributes<HTMLElement>
```

Comment:
Why call this editDisabled instead of disabled? If you want to specify what's disabled, it's better to use something like editButtonDisabled. But for now I would prefer if this is just called `disabled` instead of `editDisabled`.